docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -1049,6 +1049,14 @@ adjusted to a lower value. The minimum value is `10` and the maximum is `65,536`
 The default value is `500`.
 end::transform-settings-max-page-search-size[]
 
+tag::transform-settings-num-failure-retries[]
+Defines the number of retries on a recoverable failure before giving up.
```

Comment:
Small adjustment, so the description is more in line with `xpack.transform.num_transform_failure_retries`.
```suggestion
Defines the number of retries on a recoverable failure before the {transform} task is 
marked as `failed`.
```

Reply:
Done.